gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,504 @@
+.. license-header
+  SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+  SPDX-License-Identifier: Apache-2.0
+
+############################################################
+Install the NVIDIA GPU DRA Driver and Configure IMEX Support
+############################################################
+
+THe NVIDIA GPU DRA Driver is an additional component you can install after the GPU Operator that enables you to use the Kubernetes DRA feature to define IMEX channel resources that are managed by Kubernetes.

```

Comment:
What about:
```suggestion
The NVIDIA GPU DRA Driver is an additional component you can install alongside the GPU Operator that enables you to use the Kubernetes DRA feature to support Multi-Node NVLink in distributed applications.

```
We may also want to link to the Kubernetes DRA documentation? https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/

Reply:
updated